Screenbox/Controls/AudioTrackSubtitlePicker.xaml

```diff
@@ -23,7 +23,8 @@
                     <TextBlock
                         Margin="15,0,0,0"
```

Comment:
Can you try "16,0,0,0"?

Reply:
Here is 16,0,0,0. I think it aligns better.

![image](https://github.com/huynhsontung/Screenbox/assets/31434093/adf6bb1f-7487-43be-9360-94bb8b3261ee)

Reply:
Updated margin to "16,0,0,0" and pushed to main.